Cannot find module on 3.1.0

```
`error - Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/m/Documents/Packagetome/node_modules/use-breakpoint/dist/esm/createMediaQueries' imported from /home/m/Documents/Packagetome/node_modules/use-breakpoint/dist/esm/getCSSMediaQueries.js
    at new NodeError (node:internal/errors:400:5)
    at finalizeResolution (node:internal/modules/esm/resolve:260:11)
    at moduleResolve (node:internal/modules/esm/resolve:879:10)
    at defaultResolve (node:internal/modules/esm/resolve:1087:11)
    at nextResolve (node:internal/modules/esm/loader:161:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:834:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:415:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
    at link (node:internal/modules/esm/module_job:76:36) {
  digest: undefined
} `
```

```
`{
  packagetome: '0.1.0',
  npm: '9.2.0',
  node: '19.3.0',
  v8: '10.8.168.21-node.8',
  uv: '1.44.2',
  zlib: '1.2.13',
  brotli: '1.0.9',
  ares: '1.18.1',
  modules: '111',
  nghttp2: '1.51.0',
  napi: '8',
  llhttp: '8.1.0',
  uvwasi: '0.0.13',
  openssl: '3.0.7+quic',
  cldr: '42.0',
  icu: '72.1',
  tz: '2022f',
  unicode: '15.0',
  ngtcp2: '0.8.1',
  nghttp3: '0.7.0'
}`
```

![image](https://github.com/iiroj/use-breakpoint/assets/25252808/bd4fe756-e0c9-405b-9f0a-eb9bf0ca6215)

styleguide gave a more verbose error message that helped track down the issue;

```
Module not found: Error: Can't resolve './createMediaQueries' in '/[snip]/.pnpm/use-breakpoint@3.1.0_react-dom@18.2.0_react@18.2.0/node_modules/use-breakpoint/dist/esm'
Did you mean 'createMediaQueries.js'?
BREAKING CHANGE: The request './createMediaQueries' failed to resolve only because it was resolved as fully specified
(probably because the origin is strict EcmaScript Module, e. g. a module with javascript mimetype, a '*.mjs' file, or a '*.js' file where the package.json contains '"type": "module"').
The extension in the request is mandatory for it to be fully specified.
Add the extension to the request.
```

--- a/src/getCSSMediaQueries.ts
+++ b/src/getCSSMediaQueries.ts
@@ -1,5 +1,5 @@
-import type { Config, MediaQuery } from './createMediaQueries'
-import createMediaQueries from './createMediaQueries'
+import type { Config } from './createMediaQueries.js'
+import createMediaQueries from './createMediaQueries.js'
 
 /**
  * Get CSS native media queries for given breakpoint configuration.

